getting-started/setup-building.rst

```diff
@@ -466,10 +457,10 @@ and ``make``::
     $ make -s -j2
 
 There will sometimes be optional modules added for a new release which
-won't yet be identified in the OS level build dependencies. In those cases,
+won't yet be identified in the OS-level build dependencies. In those cases,
 just ask for assistance on the core-mentorship list.
 
-Explaining how to build optional dependencies on a UNIX based system without
+Explaining how to build optional dependencies on a Unix-based system without
```

Comment:
It's referred to the uppercase UNIX the other two times it is mentioned on the page, so IMO it would be better to either change all of them or none of them in this PR, rather than be inconsistent.

Reply:
Yep, updated the other two to Unix.

Unix is in our style guide: https://devguide.python.org/documentation/style-guide/#capitalization